config/schema.sql

```diff
@@ -103,6 +103,9 @@ CREATE TABLE oauth_data (
     code text NOT NULL
 );
 
+CREATE UNIQUE INDEX on oauth_data (code);
+ALTER TABLE oauth_data ADD CONSTRAINT unique_code UNIQUE (code);
```

Comment:
I think from looking at the PostgresSQL documentation, it seems that a `UNIQUE CONSTRAINT` under the hood is the same as a `UNIQUE INDEX` (i.e. creates a B-tree index), but it has some additional checking for uniqueness, and it appears the two have some differences in terms of features/options.

Regardless, an `INDEX` is probably the better option since we're actually going to be using it as an index (also makes it clearer what the intended usage is for); I'll go ahead and get rid of the `CONSTRAINT` portion.